docs/source/tutorials/ScadaInterfaceExample.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
As far as I can see, there is no particularly very sensitive information here, but should we double check with AC/SC?